### DISCUSSION TITLE
Tcases with oneOf multiple schemas

### DISCUSSION THREAD

**Discussion by aeringis** [Cornutum/tcases#254] [Q&A] [answered]
Hello, 

I was testing to see how tcases handles the use of oneOf in openapi. Based on this example spec, generating tests with tcases-api-test is possible and functions correctly, but does not do so in examples mode. If run with source set to examples, tcases is unable to generate any test data of its own, even if no example data is provided which I had thought should function equivalently to  running tcases normally. Is this the expected behavior? 

https://github.com/Cornutum/tcases/blob/master/tcases-openapi/README.md#openapi-tips

Based on reading the tips in above, I believe it should be possible to have tcases work with the spec layout provided and have examples for some of the properties in the schema section and have tcases generate random values for others 

<img width="693" alt="Screen Shot 2022-11-09 at 2 08 55 PM" src="https://user-images.githubusercontent.com/53023246/200932614-9632816e-7934-4905-886f-23f9668acc60.png">

**Comment by kerrykimbrough** [comment]
Yes, this should work. Creating an issue to resolve this problem.

**Comment by kerrykimbrough** [comment, marked as answer]
See issue #255
